src/Evolution/Ringdown/StrahlkorperCoefsInRingdownDistortedFrame.hpp

```diff
@@ -16,23 +16,33 @@
  */
 namespace evolution::Ringdown {
 /*!
- * \brief Transform `Strahlkorper` coefs to ringdown distorted frame.
+ * \brief Transforms `Strahlkorper` coefs to ringdown distorted frame and tracks
+ * the center of mass.
  *
  * \details Reads Strahlkorper coefficients (assumed to be in the inertial
  * frame) from a file, then transforms them into the
- * ringdown distorted frame defined by the expansion and rotation maps
- * specified by `exp_func_and_2_derivs`, `exp_outer_bdry_func_and_2_derivs`,
- * and `rot_func_and_2_derivs`, which correspond to the ringdown frame's
- * expansion and rotation maps at the time given by `match_time`, and by
- * `settling_timescale`, the timescale for the maps to settle to constant
- * values. Only Strahlkorpers within `requested_number_of_times_from_end` times
- * from the final time are returned. This function is used to transition
- * from inspiral to ringdown; in this case, the inertial-frame Strahlkorper
- * is the common apparent horizon from a binary-black-hole inspiral;
- * the ringdown-distorted-frame coefficients are used to initialize
- * the shape map for the ringdown domain.
+ * ringdown distorted frame defined by the expansion, rotation, and translation
+ * maps specified by `exp_func_and_2_derivs`,
+ * `exp_outer_bdry_func_and_2_derivs`, `rot_func_and_2_derivs`, and
+ * `trans_func_and_2_derivs`. The expansion and rotation functions of time
+ * correspond to the ringdown frame's expansion and rotation maps at the time
+ * given by `match_time`, and by `settling_timescale`, the timescale for the
+ * maps to settle to constant values. The translation function of time supplied
+ * does not correspond to the ringdown frame's translation function of time. The
+ * ringdown's translation function of time needs to be built by tracking the
+ * position of the center of mass at multiple times. Only Strahlkorpers and
+ * center of mass points within `requested_number_of_times_from_end` times from
+ * the final time are returned. This function is used to transition from
+ * inspiral to ringdown; in this case, the inertial-frame Strahlkorper is the
+ * common apparent horizon from a binary-black-hole inspiral; the
+ * ringdown-distorted-frame coefficients are used to initialize the shape map
+ * for the ringdown domain. The center of mass points are used to initialize the
+ * translation map for the ringdown domain.
```

Comment:
What is the translation function of time that is passed into this function?  Is it the translation function of time of the inspiral?  (You say what it is not, but not what it is).

And what are "center of mass points"?   Does the center of mass actually matter here?